comptest/web/models.py

```diff
@@ -24,21 +29,31 @@ class Project(models.Model):
         default=None,
         related_name="projects",
     )
+    gh_repo = models.URLField(max_length=1024, default="https://example.com")
```

Comment:
This should be something like a “metadata” json field, as different challenges would want different metadata to be collected. And then the list of metadata we wanna collect should be specified as a django setting, perhaps as jsonschema (or just a list for now to meet deadlines)

Reply:
Resolved in https://github.com/2i2c-org/unnamed-thingity-thing/pull/123/commits/4de76606c2d4a3b1a78ed885a453d96d43a4f8ae